### PR TITLE
uplode_t5

### PR DESCRIPTION
**Pull request by asifnaseebi**

### CODE REVIEW

File: utils/reader.py [hunk 1]
```diff
@@ -10,3 +10,16 @@ def file_read():
         rows.append(row)
 
     return rows
+
+
+
+def file_t5_read():
```
Comment (Hamma111) [inline utils/reader.py:16]: use one function to read these files

have your function which takes on paramater of file_path and return the files contents from that function. should be used like this
```python
file1_content = read_file(".../f1.csv")
file2_content = read_file(".../f2.csv")

**Review by Hamma111 — CHANGES_REQUESTED** [review]

File: task/t5.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
+from utils.reader import file_t5_read
+
+files_values = file_t5_read()
+for files_value in files_values:
+    date = files_value.split(",")[1]
+    events = files_value.split(",")[-2]
+    if events in ["Rain", "Snow","Rain-Snow"]:
```
Comment (Hamma111) [inline task/t5.py:7]: good. but study `if x in y`
https://realpython.com/python-conditional-statements/#:~:text=%3E%3E%3E%20if%20%27aul%27%20in%20%27grault%27%3A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%23%20Truthy

File: utils/reader.py (hunk 1 above)
Comment (asifnaseebi) [reply utils/reader.py:16]: ok I am trying

File: task/t2.py [hunk 3]
```diff
@@ -1,11 +1,10 @@
 from utils.reader import file_read
 
-
-files_values = file_read()
-for row in files_values:
-    date = row.split(",")[0]
-    max_tmp = row.split(",")[1]
-    min_tmp = row.split(",")[3]
+path = file_read( "/home/asif/Desktop/project/barq_weather_task/files/f1.csv")
```
Comment (Hamma111) [inline task/t2.py:3]: path is not a good variable name, asif. these are file values or file contents. i expect better names from you

**Review by Hamma111 — CHANGES_REQUESTED** [review]

**Pull request merged**